packages/components/src/resizable-box/README.md

```diff
@@ -53,3 +53,11 @@ const Edit = ( props ) => {
 	);
 }
 ```
+
+### Props
```

Comment:
By the way, this is why I like when you abstract away external code, you get the freedom to tailor it to your needs 👍

Reply:
Yay! Thank you 🙌